src/app/static/src/js/components/runWorkflow/runWorkflowSummary.vue

```diff
@@ -1,48 +1,186 @@
 <template>
     <div>
         <h2 id="summary-header">Summary</h2>
-        <span>Your workflow contains {{ reportCount }}:</span>
-        <ul class="styled">
-            <li v-for="report in workflowMetadata.reports">
-                {{ report.name }}
-            </li>
-        </ul>
+        <div id="summary-warning" class="row mt-3" v-if="hasMissingDependencies">
+            <div class="col-auto">
+                <alert-triangle-icon size="2x" stroke="red" class="custom-class"/>
+            </div>
+            <div class="col-auto">
+                <span class="d-inline-block pb-2"> Some reports depend on the latest version of other reports that are not included in your workflow:</span>
+                <div v-for="(missing_dependencies, report) in dependencies.missing_dependencies">
+                    <span v-if="missing_dependencies.length" class="font-weight-bold"> {{ report }}</span>
+                    <ul v-for="missing_dependency in missing_dependencies" class="styled">
+                        <li>{{ missing_dependency }}</li>
+                    </ul>
+                </div>
+            </div>
+            <hr>
+        </div>
+        <div id="summary-timeline" class="mt-3">
+            <div class="row">
+                <div class="col-12">
+                    <div class="summary">
+                        <div v-for="report in workflowMetadata.reports" class="single-timeline-area">
+                            <div class="timeline-date wow fadeInLeft" data-wow-delay="0.1s"
+                                 style="visibility: visible; animation-delay: 0.1s; animation-name: fadeInLeft;">
```

Comment:
I couldn't detect any animation happening, and I don't think it would be very helpful anyway, so maybe we don't need any of these animation styles.